contrib/python-maturin/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "python-maturin"
```

Comment:
on a second thought, split specifically this into its own PR and leave the others in this one

Reply:
while at it, this does not bother to properly set features (`--features=full,native-tls,password-storage`), which need to be set (because we don't want to build with rustls)